library/sonarqube

```diff
@@ -1,22 +1,22 @@
-Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
-             Julien Lancelot <julien.lancelot@sonarsource.com> (@julienlancelot),
-             Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
+Maintainers: Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
-             Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
+             Tobias Trabelsi <tobias.trabelsi@sonarsource.com> (@tobias-trabelsi-sonarsource),
+             Lukasz Jarocki <lukasz.jarocki@sonarsource.com> (@lukasz-jarocki-sonarsource)
+
```

Comment:
I think this new blank line is confusing the parser, so that it thinks the next block is not part of the "global" config.

Reply:
Thank you. It looks better now.